dist/wavesurfer.js

```diff
@@ -333,18 +333,13 @@ var WaveSurfer = {
      */
     load: function (url, peaks) {
         switch (this.params.backend) {
-            case 'WebAudio': return this.loadBuffer(url);
+            case 'WebAudio': return this.loadWebAudio(url, peaks);
             case 'MediaElement': return this.loadMediaElement(url, peaks);
         }
     },
 
-    /**
-     * Loads audio using Web Audio buffer backend.
-     */
-    loadBuffer: function (url) {
-        this.empty();
-        // load via XHR and render all at once
-        return this.getArrayBuffer(url, this.loadArrayBuffer.bind(this));
+    loadWebAudio: function(url, peaks) {
+        return this.loadBuffer(url, peaks);
```

Comment:
What is the point of this function if it just calls another function with the same signature?